### PR TITLE
col test added

### PR DESCRIPTION
**Pull request by newproject25**
hey.what do you think?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,5 +18,7 @@ <h4>h4 added</h4>
         </ol>
         
     </nav>
+
```
Comment (sajjadmohammadi23) [inline index.html:21]: whats that

Comment (sajjadmohammadi23) [reply index.html:21]: th

File: index.html [hunk 2]
```diff
@@ -18,5 +18,7 @@ <h4>h4 added</h4>
...
+    <p>this is a col test</p>
```
Comment (newproject25) [inline index.html:22]: just a test

Comment (sajjadmohammadi23) [reply index.html:22]: ok

**Comment by sajjadmohammadi23** [comment]
done

**Pull request merged**